components/main_menu/index.jsx

```diff
@@ -56,9 +60,11 @@ function mapStateToProps(state) {
         appDownloadLink,
         enableCommands,
         enableCustomEmoji,
+        canManageIntegrations,
         enableIncomingWebhooks,
         enableOAuthServiceProvider,
         enableOutgoingWebhooks,
+        canManageSystemBots,
```

Comment:
Need `canManageSystemBots` to act as the "are bots enabled" config flag of sorts.